doc/rtd/explanation/hardening.rst

```diff
@@ -0,0 +1,80 @@
+Security Hardening
+******************
+
+Cloud-init's use case is automating cloud instance initialization, with support
+across distributions and platforms. There is a myriad of ways to harden this
+space.
```

Comment:
That second sentence is vague. "harden this space" doesn't hold much meaning. Let's correct this to:
```suggestion
across distributions and platforms. There are a myriad of ways to improve the
security posture of a cloud-init configured machine.
```